packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -106,14 +106,23 @@ abstract contract FundingRateApplier is FeePayer {
         address _collateralAddress,
         address _finderAddress,
         address _configStoreAddress,
+        FixedPoint.Unsigned memory _tokenScaling,
         address _timerAddress
     ) public FeePayer(_collateralAddress, _finderAddress, _timerAddress) {
         uint256 currentTime = getCurrentTime();
         fundingRate.updateTime = currentTime;
         fundingRate.applicationTime = currentTime;
 
-        // Seed the cumulative multiplier as 1, from which it will be scaled as funding rates are applied over time.
-        fundingRate.cumulativeMultiplier = FixedPoint.fromUnscaledUint(1);
+        // Seed the cumulative multiplier with the token scaling, from which it will be scaled as funding rates are
+        // applied over time.
+        FixedPoint.Unsigned memory minScaling = FixedPoint.Unsigned(1e8); // 1e-10
```

Comment:
OOC why validate this within the perp as opposed to in the PerpCreator?

Reply:
No super strong reason. We could move it out for the same reasons we moved out the validation of other parameters, like the synthetic token. Thoughts?

Reply:
Bytecode definitely is a reason to move it out, but I was thinking more for consistency. The creator does some validation on Perp constructor params while it seems that our stance on the Perp itself is not to validate any params. Thoughts?

Reply:
Agreed, consistency is another good reason! We'll also be able to add back the require message!

Reply:
Will update

Reply:
Done.